README.md

```diff
@@ -93,15 +93,15 @@ If you are not on the keyring, enable the Google Sheets and Google Drive APIs in
 create a service account to access them. [This blog post](https://www.twilio.com/blog/2017/02/an-easy-way-to-read-and-write-to-a-google-spreadsheet-in-python.html)
 provides a helpful walkthrough! Be sure to save your credentials in
 `sfm_pc/management/commands/credentials.json` and give your service account
-access to the correct files before you run `import_google_docs`.
+access to the correct files before you run `download_country_data`.
 
 Finally, import entity data from Google Drive:
 
     # Import entity data from default files
-    docker-compose run --rm app make import_google_docs
+    docker-compose run --rm app make import_country_data
 
     # Import entity data from specified files
-    docker-compose run --rm app python manage.py import_google_doc --doc_id ${SOME_ID} --source_doc_id ${SOME_ID} --location_doc_id ${SOME_ID}
+    docker-compose run --rm app python manage.py download_country_data --entity_doc_id ${SOME_ID} --sources_doc_id ${SOME_ID} --location_doc_id ${SOME_ID} --country_code ${SOME_CODE} --parent_directory ${DIRECTORY}
```

Comment:
Is it possible to import without downloading, or should we instruct folks to always run download, then import?

Reply:
Ah, good catch, realizing this might be bad information — I'll rework. Do you think we need this "# Import entity data from specified files" instruction? That information exists in the "Importing the data" section. There's a little bit of overlap in information, but I see a little bit of value in keeping the section here for setting up local development while still having a separate "importing the data" section that gets at a lower level.

Reply:
I'll take redundancy for the sake of ease. I think I'd rework like:

```bash
# Download country data listed in fixtures/import_docket.csv
... download command ...

# Import downloaded country data
... import command ...
```

Thoughts?

Reply:
@hancush Agreed about the sake of ease. What about removing the individual steps and just adding the command for the import_docket_import make recipe, since that recipe calls both commands and would import the data for local development? (I think that would achieve that, right?) When I first started on this project, I found it kind of confusing to have the 

```
docker-compose run --rm app python manage.py import_google_doc --doc_id ${SOME_ID} --source_doc_id ${SOME_ID} --location_doc_id ${SOME_ID}
```

command spelled out as an instruction rather than an example, especially since it's explained later in the doc. There are a bunch of different ways to import the data in this readme, so it's helpful for me to have an explicit, one way instruction for setting up local development. And then later include more granular stuff (examples and different ways to do it). 

I think it's a question of what this section should contain. It's called "Load data via management commands" and the readme in the master branch says to run `import_google_docs` make recipe but I don't see that anymore. Do we want developers to run the commands for each entity by hand, or run individual make recipes, or an instruction with an automated make process so they're setup to develop? And do we expect a dev to download/import all of the data or just work with a little bit of it?

Reply:
Good thought, @smcalilly. Let's do `make import_docket_import`, and point to docs further down on loading a subset of data, if they want to.

Reply:
If we want the local dev setup to only include a snippet of the data then I think your suggestion makes the most sense.

Reply:
@smcalilly I endow you with the power to make a decision here, haha.

Reply:
I think you had it right after seeing that section as a whole, changed here 4336c21c